modules/security-hosts-vms-vs-containers.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-hosts-vms.adoc
+
+[id="security-hosts-vms-vs-containers_{context}"]
+= Virtualization versus containers
```

Comment:
Not critical, but maybe could consider "Comparing virtualization and containers" ?

Reply:
Sure.